test/plugin_functional/config.js

```diff
@@ -32,7 +32,7 @@ export default async function ({ readConfigFile }) {
       require.resolve('./test_suites/app_plugins'),
       require.resolve('./test_suites/custom_visualizations'),
       require.resolve('./test_suites/embedding_visualizations'),
-      require.resolve('./test_suites/panel_actions'),
+      //  require.resolve('./test_suites/panel_actions'),
```

Comment:
Move to next PR?
Uncomment?

Reply:
yea, I'll clean this up.  Was running into ci issues with my sample plugin, so I have some lingering code in here to try to track it down (I think it was the scss files... really weird).